net/discovery.go

```diff
@@ -28,7 +28,7 @@ type discovery struct {
 	dht         *dual.DHT
 	h           libp2phost.Host
 	rd          *libp2prouting.RoutingDiscovery
-	provides    []types.ProvidesCoin // set to a single item slice of XMR when we make an offer
+	provides    []coins.ProvidesCoin // set to a single item slice of XMR when we make an offer
```

Comment:
I need to update this to be `[]string` lol. will do when I move this to a separate repo